python/src/trezorlib/cardano.py

```diff
@@ -108,16 +103,85 @@ def create_input(input) -> messages.CardanoTxInputType:
 
 
 def create_output(output) -> messages.CardanoTxOutputType:
```

Comment:
Where is such a function actually needed? In CLI? That's not actually used in production by real users and is not run on the device AFAIK. I would prefer to keep such code separate.

Reply:
It's used by the CLI and also by the tests. Other cryptocurrencies have helper functions in this file as well, however some have separated them from the "Client functions" with a comment:
`# ====== Client functions ====== #`
Maybe that would help the separation a tiny bit. I don't really want to create a new file for this.  I would do that in a separate PR anyways.

Reply:
I understand it a bit better now, the whole file is dedicated to manipulating protobuf messages. It's OK to keep it all together. I like using `# ====== Client functions ====== #`, seems like a reasonable convention.